src/omega.rs

```diff
@@ -0,0 +1,67 @@
+/// Lambert W function.
+pub trait Omega
+where
+    Self: Sized,
+{
+    /// Compute the real-valued parts of the princial branch of the Lambert W function.
+    ///
+    /// This function is the inverse of the function
+    ///
+    /// ```math
+    /// f(w) = we^w
+    /// ```
+    ///
+    /// when w >= -1.
+    ///
+    /// ## Example
+    ///
+    /// ```
+    /// use special::Omega;
+    ///
+    /// assert!((1.0.lambert_w0() - 0.5671432904097838).abs() < 1e-15);
+    /// ```
+    ///
+    /// ## References
+    ///
+    /// - T. Fukushima, Precise and fast computation of Lambert W function
+    /// by piecewise minimax rational function approximation with variable transformation,
+    /// To be submitted, 2020, <https://doi.org/10.13140/RG.2.2.30264.37128>.  
+    /// - The [`lambert_w`] crate.
```

Comment:
This I would move up into the main description as it is done here:

https://github.com/stainless-steel/special/blob/main/src/gamma.rs#L43